server/uploader.js

```diff
@@ -30,23 +30,36 @@ module.exports = {
    */
   uploadFile: function(desc, file, prefix, bigWigToWigAddr) {
     var source = fs.createReadStream(file.path);
-    var dest = fs.createWriteStream(prefix + desc.name);
+    if (fs.existsSync(prefix + desc.fileName)) {
+      console.log('wiggle file already exists');
+      return {
+        success: false,
```

Comment:
We plan to make error handling by "success" flag obsolete. Return error object on failure and empty object on success (if no data need to be returned).
```js
return {
  error: {
    type: 'wiggleExists',
    message: 'wiggle file already exists for current name'
  }
};
```